src/Sensors/Sentinel1/Sentinel1Types/Sentinel1Types.jl

```diff
@@ -11,4 +11,47 @@ end
 ## implement interface
 get_metadata(image::Sentinel1SLC) = image.metadata;
 get_data(image::Sentinel1SLC) = image.data;
-is_deramped(image::Sentinel1SLC) = image.deramped;
\ No newline at end of file
+is_deramped(image::Sentinel1SLC) = image.deramped;
+get_time_range(image::Sentinel1SLC) = (image.metadata.header.start_time, image.metadata.header.stop_time)
+
+
+function get_window(image::Sentinel1SLC)
+    index_1_start = image.index_start[1]
+    index_1_end = size(image.data)[1] + image.index_start[1] - 1
+
+    index_2_start = image.index_start[2]
+    index_2_end = size(image.data)[2] + image.index_start[2] - 1
+
+    return [[index_1_start, index_1_end],[index_2_start, index_2_end]]
+end
+
+function get_burst_numbers(image::Sentinel1SLC)
```

Comment:
isn't this more like get_burst_lines()?

Reply:
ah no, right I understand. Maybe a small docstring would be useful to clarify what "numbers" means here